docs/user_guide/getting_started/minimal_script.rst

```diff
@@ -0,0 +1,115 @@
+.. currentmodule:: globus_sdk
+
+.. _minimal_script_tutorial:
+
+Creating A Minimal Script
+=========================
+
+This is a basic tutorial in the use of the Globus SDK.
+
+You can jump right in by using the ``CLIENT_ID`` seen in the example code blocks below!
+That is the ID of the tutorial client, which lets you get started quickly and easily.
+
+When you are ready to create your own application, follow :ref:`the tutorial on
+registering a Native App <tutorial_register_app>` and use its ``CLIENT_ID`` in
+the rest of the tutorial to get your own app setup.
+
+For readers who prefer to start with complete working examples, jump ahead to the
+:ref:`example scripts <minimal_script_complete_examples>` at the end before reviewing
+the doc.
+
+Define your App Object
+----------------------
+
+Accessing Globus APIs as a user requires that you login to your new app and get
+it tokens, credentials providing access the service.
+
+The SDK provides a construct which represents an application. A
+:class:`GlobusApp` is an object which can respond to requests for new or
+existing tokens and store those tokens (by default, in ``~/.globus/app/``).
+
+:class:`UserApp` is the type of :class:`GlobusApp` used for human user,
+login-driven scenarios -- this is always the right type of application object
+to use when you want to interact with services using your own account.
+
+Start by defining an application object using :class:`UserApp`:
+
+.. code-block:: python
+
+    import globus_sdk
+
+    # this is the tutorial client ID
+    # replace this string with your ID for production use
+    CLIENT_ID = "61338d24-54d5-408f-a10d-66c06b59f6d2"
+
+    # create your app
+    my_app = globus_sdk.UserApp("my-user-app", client_id=CLIENT_ID)
+
+
+.. note::
+
+    The default behavior for a :class:`UserApp` is to do a CLI-based login flow.
+    This behavior can be disabled or customized in numerous ways.
+
+    For the full menu of options, look at the documentation about :ref:`Using a
+    GlobusApp! <using_globus_app>`
+
+Access the APIs via Clients
+---------------------------
+
+Once you have an app defined, you can use it with client objects to access
+various Globus APIs.
+
+When you attempt to interact with a service, the app will automatically prompt
+you to login.
```

Comment:
```suggestion
When you attempt to interact with a service using an app-bound service client, the app will
automatically prompt your user to login if valid credentials are unavailable.
```

Reply:
Mostly :+1: , but I prefer "prompt you to login" over "prompt your user", since the expected usage in this guide is that you're writing a script for your own use.